### PR TITLE
Ember Times 208

### PR DESCRIPTION
**Pull request by jaredgalanis**
## Released packages (WONT DO)
- v2 add-on blueprint support from ember-cli, mirage v3 , ember-command v2 , ember-workshop update , ember-resize-modifier 0.7 ,qunit-dom v3 , ember-template-imports v4 

## Internal
- [ ] Typescript guides https://discord.com/channels/480462759797063690/480499624663056390/1166059225982717963
- [ ] https://blog.emberjs.com/ember-data-4-12-special-release-update-2023-09-22
- [ ] Call for contributors to help with EmberData Request Service Cheat Sheet - https://github.com/ember-learn/ember-data-request-service-cheat-sheet/issues/1
- [ ] AI bot on discord (maybe?) https://discord.com/channels/480462759797063690/480499624663056390/1164300654647455804

## Videos
- [x] EmberFest videos https://www.youtube.com/@emberfest @MinThaMie 🖊️ 
- [x] Complete deep dive from Dan Freeman and Chris Krycho (all 6 episodes) https://www.youtube.com/watch?v=9RjaREOmFAA&list=PLelyiwKWHHApkoeXQjwKPHPAHgKXZyl3t&index=4 @MinThaMie 🖊️ 

- [x] EmberEurope Q4 2023: https://www.meetup.com/ember-europe/events/296967770/ @MinThaMie 🖊️ 

## Blogposts
- [x] Troubleshooting CI test failures @MinThaMie 🖊️ https://discord.com/channels/480462759797063690/480499624663056390/1164300654647455804
- [x] Context leaking in tests @MinThaMie 🖊️ https://discord.com/channels/480462759797063690/480499624663056390/1164940344463466598
- [x] What to use instead of ember/string @MinThaMie 🖊️ https://discord.com/channels/480462759797063690/480499624663056390/1164947641252196382
- [ ] Update on the Embroider Initiative https://discord.com/channels/480462759797063690/480499624663056390/1166278782752129076
- [x] Setting tailwind up @MinThaMie 🖊️ https://discord.com/channels/480462759797063690/480499624663056390/1166473027123425411
- [ ] https://dev.to/sukima/why-emberjs-is-designed-for-enterprise-software-development-6le

### CODE REVIEW

**Comment by jenweber** [comment]
I added a topic for a call for contributors in https://github.com/ember-learn/ember-data-request-service-cheat-sheet/issues/1

**Comment by IgnaceMaes** [comment]
EmberEurope Q4 2023 was just announced on Meetup, so might be interesting to refer to as well: https://www.meetup.com/ember-europe/events/296967770/

**Comment by chancancode** [comment]
https://github.com/chancancode/ember-polaris-service

ping me on discord if more details are needed!

File: content/the-ember-times-issue-208.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+---
+
+## [🇪🇺 Ember Europe: Update on Embroider initiative](https://www.youtube.com/watch?v=gPMzrQzryZI)
+
+[Chris Manson (@mansona)](https://github.com/mansona) provided an update on the [Embroider Initiative](https://mainmatter.com/embroider-initiative/)'s progress during his talk at Ember Europe’s final 2023 Meetup. 
+
+He addressed the [Vite](https://vitejs.dev) support and demonstrated an Ember app on Vite, explaining the importance of using ESM-compatible dependencies. He also discussed the challenges faced during the integration process, such as add-ons not being fully converted to ESM. 
+
+Chris introduced the upcoming [unstable Embroider release](https://github.com/embroider-build/embroider/releases) and encouraged users to remain optimistic despite encountering issues due to cache busting by showcasing debugging techniques and the advantages of Vite's network tab.
```
Comment (mansona) [inline content/the-ember-times-issue-208.md]:
```suggestion
Chris introduced what will be in the [next major release of Embroider](https://github.com/embroider-build/embroider/pull/1701) and encouraged users to remain optimistic despite encountering issues due to cache busting by showcasing debugging techniques and the advantages of Vite's network tab.
```

**Pull request merged**